xml_inscriptions/metadata_only/MO.Col.UM.MAA.L.59.18.xml

```diff
@@ -83,22 +88,36 @@
                         </summary>
 
                         <origin>
-                         
-                         
+                            <date evidence="#history" notBefore="0251" notAfter="0253">251-253 CE</date>
+                            <placeName ref="unknown">unknown</placeName>
                         </origin>
+                        
+                        <provenance type="transferred" subtype="given">
+                            <p>Gift of <persName role="donor">Cornelius Ruxton Love, Jr.</persName> to the <orgName role="recipient">Museum of Art and Archaeology</orgName> of the <orgName>University of Missouri</orgName>.</p>
+                        </provenance>
 
 
                     </history>
                 </msDesc>
                 <!--  *************************************  <listBibl>  *********************************  -->
                 <listBibl>
+                    
                     <bibl>
-                      
-                      
+                        <ref type="external"
+                            target="http://maacollections.missouri.edu/ArgusNET/Portal.aspx?lang=en-US&amp;g_AAEM=59.18"
+                            >Museum of Art and Archeology (Web Entry)</ref>
                     </bibl>
                     <bibl>
-                        <ref type="external" target="http://maacollections.missouri.edu/ArgusNET/Portal.aspx?lang=en-US">External Reference</ref>
+                        <ref type="external"
```

Comment:
These are linking directly to bibliography house on the MAA website, yes? This is fine, though generally we'd rather keep bibliography of this sort internally. We can talk about it at our meeting, though.